bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -435,7 +499,12 @@ async def post_review(self, ctx: Context, user_id: int) -> None:
     @Cog.listener()
     async def on_member_ban(self, guild: Guild, user: MemberOrUser) -> None:
         """Remove `user` from the talent pool after they are banned."""
-        await self.end_nomination(user.id, "User was banned.")
+        if await self.end_nomination(user.id, "Automatic removal: User was banned"):
+            nomination_discussion = self.bot.get_channel(Channels.nomination_discussion)
```

Comment:
Same thing about the `get_channel` here, it might even start to be redudnant :/

Reply:
https://github.com/python-discord/bot/pull/2418/commits/076f0714a43006bff7c5d39aa80f898ab3254c26